docs/source/gcloud.rst

```diff
@@ -11,6 +11,14 @@ Google Compute Engine (gce) instances are provisioned using this resource.
 * :docs1.5:`Topology Example <workspace/topologies/gce-new.yml>`
 * `Ansible module <http://docs.ansible.com/ansible/latest/gce_module.html>`_
 
+gcloud_gce_eip
+--------------
+
+Google Compute enginer external IP (gce_eip) are provisioned using this resource.
+
+* :docs1.5:`Topology Example <workspace/topologies/gce-eip.yml>`
```

Comment:
I know that the file I mentioned above is referenced here.  Is there a way to do this that uses `workspaces` instead of `workspace`?  Even if we just create a general topologies directory in `workspaces` for the topologies used in the docs to avoid breaking up the PinFiles

Reply:
@14rcole Sure will try to move everything to workspaces folder with other PRs too